### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by daemyung**
## 🙏 Describe the pull request

README.md is updated.

## ✅ Checklist

- [x] Code follows the project's coding conventions and style.
- [x] Tests have been added or updated to cover the changes.
- [x] Documentation has been updated, if necessary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,12 @@ detail.
 bash install_package.sh
 ```
 
+You can also install Trident using pip.
+
+```bash
+pip install git+https://github.com/kakaobrain/trident.git
```
Comment (daemyung) [inline README.md:26]: @ansteve pip3일 필요는 없겠죠?

Comment (kakao-steve-ai) [reply README.md:26]: 요즘 Python 2.x 거의 안사용하는거 같아서 그냥 pip 이라고 써도 괜찮을거 같습니다.

**Review by kakao-steve-ai — APPROVED** [review]
LGTM

**Review by mejai1206 — APPROVED** [review]

**Comment by mejai1206** [comment]
LGTM

**Pull request merged**